ci-scripts/dockerfiles/Tmod4.dockerfile

```diff
@@ -0,0 +1,41 @@
+#
+# Execute this from the top-level ReFrame source directory
+#
+
+FROM ubuntu:20.04
+
+ENV TZ=Europe/Zurich
+ENV DEBIAN_FRONTEND=noninteractive
+ENV _TMOD_VER=4.6.0
+WORKDIR /root
+
+# ReFrame requirements
+RUN \
+  apt-get -y update && \
+  apt-get -y install gcc && \
+  apt-get -y install make && \
+  apt-get -y install git && \
+  apt-get -y install python3
+
+# Required utilities
+RUN apt-get -y install wget
+
+# Install Tmod4
+RUN \
+  apt-get -y install autoconf && \
+  apt-get -y install tcl-dev && \
+  wget -q https://github.com/cea-hpc/modules/archive/v${_TMOD_VER}.tar.gz -O tmod.tar.gz && \
+  tar xzf tmod.tar.gz && \
+  cd modules-${_TMOD_VER} && \
+  ./configure && make install
+
+ENV BASH_ENV=/usr/local/Modules/init/profile.sh
+
+# Install ReFrame from the current directory
+COPY . /root/reframe/
+RUN \
+  cd reframe \
```

Comment:
```suggestion
  cd reframe && \
```

Reply:
This one is also funny, because it worked...

Reply:
Ok, I've seen the whole file. It all makes sense now! It is due to this that I didn't see that `python3-pip` was required and it was because of the `COPY` that it was working without the `bootstrap.sh` :-) I'll fix that.

Reply:
It works but it does not execute the bootstrap script.